sdk/analytics/src/main/java/com/klaviyo/analytics/model/ProfileKey.kt

```diff
@@ -13,6 +13,7 @@ sealed class ProfileKey(name: String) : Keyword(name) {
     internal object ANONYMOUS_ID : ProfileKey("anonymous_id")
     internal object PUSH_TOKEN : ProfileKey("push_token")
     internal object PUSH_STATE : ProfileKey("push_state")
+    internal object API_KEY : ProfileKey("api_key")
```

Comment:
Maybe just semantics to how state is organized in the Android SDK but this isn't really a _profile_ property, right? Is there potential this could be better exposed from some other place?

Reply:
Truth, could do this instead
```
internal object API_KEY : Keyword("api_key")
```